CHANGELOG.md

```diff
@@ -11,7 +11,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - chore: update opentelemetry core and contrib to v0.81.0 [#1187]
 
+### Added
+
+- feat: adding new components [#1188]:
+
+  - apachesparkreceiver
+  - azureblobreceiver
+  - azuremonitorreceiver
+  - chronyreceiver
+  - filereceiver
+  - oracledbreceiver
+  - pulsarreceiver
+  - snowflakereceiver
```

Comment:
The following components have stability level `development`, whereas we require at least Alpha:
- `apachesparkreceiver `
- `azureblobreceiver`
- `azuremonitorreceiver`
- `filereceiver`

Reply:
Then we should add it.

Reply:
Please review :)